suite-common/wallet-core/src/device/deviceReducer.ts

```diff
@@ -866,3 +866,22 @@ export const selectDeviceState = (state: DeviceRootState) => {
 
     return device?.state ?? null;
 };
+
+export const selectDeviceInstances = memoize((state: DeviceRootState) => {
```

Comment:
Just curious. Is there specific reason to memoize or not?

Reply:
getDeviceInstances returns new array that would cause rerenders